.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,24 +7,24 @@ assignees: ''
 
 ---
 
-**Describe the bug**
+### Description
 A clear and concise description of what the bug is.
 
-**To Reproduce**
+### Steps to Reproduce
 Steps to reproduce the behavior:
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
-**Expected behavior**
+### Expected behavior
 A clear and concise description of what you expected to happen.
 
-**Screenshots**
+### Screenshots
```

Comment:
There are no screenshots for a `dart` package, right? Or in very rare cases, I can't think of one of them now.